strapi/strapi
# fix "Uncaught error trying to nullify/remove a date field in Content Manager"

### What does it do?

This PR tries to fix a problem inside the Helper Plugin when we use the DateTimePicker component. 

### Why is it needed?

When you try to delete the date content in a DateTimePicker field in the Content manager you have an error.

### How to test it?

1. Create an entity schema in the Content Builder with a date+time field.  Do not make it required.
2. Go to Content Manager and use the date+time picker to select values and save it.
3. Go to Content Manager for that saved record and try to remove the datetime by clicking the "x" to the right of the field.
4. Observe nothing happens and you receive a console error.

### Related issue(s)/PR(s)

The issue related to this PR is this one https://github.com/strapi/strapi/issues/16337
